spec/facility_spec.rb

```diff
@@ -1,26 +1,156 @@
+      expect(@facility_1.services).to eq([])
+      expect(@facility_2).to be_an_instance_of(Facility)  
+      expect(@facility_2.name).to eq('DMV Northeast Branch')
+      expect(@facility_2.address).to eq('4685 Peoria Street Suite 101 Denver CO 80239')
+      expect(@facility_2.phone).to eq('(720) 865-4600')
+      expect(@facility_2.services).to eq([])
     end
   end
 
-  describe '#add service' do
+  describe '#add_service' do
     it 'can add available services' do
       expect(@facility.services).to eq([])
       @facility.add_service('New Drivers License')
       @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      @facility_2.add_service('New Drivers License')
+      @facility_2.add_service('Renew Drivers License')
+      @facility_2.add_service('Vehicle Registration')
+      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License'])
+      expect(@facility_1.services).to eq(['New Drivers License', 'Vehicle Registration'])
+      expect(@facility_2.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end   
+  end
+
+  describe '#register_vehicle' do
+    it 'can register vehicles' do
+      expect(@facility_1.register_vehicle(@cruz)).to eq(nil)
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@cruz.plate_type).to eq(nil)
+      expect(@cruz.registration_date).to eq(nil)
+      expect(@facility_1.collected_fees).to eq(0)
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(100)
+      expect(@facility_1.registered_vehicles).to eq([@cruz]) 
+
+      expect(@camaro.plate_type).to eq(nil)
+      expect(@camaro.registration_date).to eq(nil)
+      expect(@facility_1.register_vehicle(@camaro)).to eq([@cruz, @camaro])
+      expect(@camaro.plate_type).to eq(:antique)
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(125)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro]) 
+
+      expect(@bolt.plate_type).to eq(nil)
+      expect(@bolt.registration_date).to eq(nil)
+      expect(@facility_1.register_vehicle(@bolt)).to eq([@cruz, @camaro, @bolt])
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@bolt.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(325)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt]) 
+    end
+  end   
+
+  describe '#administer_written_test' do
+    it 'administer a written test' do
```

Comment:
Same here -- can you break this up into different scenarios with specific descriptions? i.e. it `can administer a written test if registrant has a permit`